lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -846,14 +849,13 @@ void TypeLoweringVisitor::visitExpr(SubfieldOp op) {
 
 // Gracefully die on subaccess operations
 void TypeLoweringVisitor::visitExpr(SubaccessOp op) {
-  op.emitError("SubaccessOp not handled.");
+  emitWarning(op.getLoc(), "SubaccessOp not handled.");
```

Comment:
Is turning this error into a warning the right thing?

Reply:
I dropped this commit from the PR, it was not a good idea